models/claims_preprocessing/encounters/final/office_visit__encounter_grain.sql

```diff
@@ -169,30 +101,29 @@ group by encounter_id
        ,max(case when scr.service_category_2 = 'observation' then 1 else 0 end) as observation_flag
        ,max(case when scr.service_category_2 = 'pharmacy' then 1 
               else 0 end) as pharmacy_flag
-    from detail_values d
+    from {{ ref('office_visits__int_detail_values') }} d
     left join {{ ref('service_category__service_category_grouper') }} scr on d.claim_id = scr.claim_id 
     and
     scr.claim_line_number = d.claim_line_number
     group by d.encounter_id
 )
 
 
-select   d.encounter_id
+select d.encounter_id
 , d.encounter_start_date
 , d.patient_data_source_id
-
-,tot.encounter_type
-,tot.encounter_group
+, tot.encounter_type
+, tot.encounter_group
 , {{ dbt.datediff("birth_date","d.encounter_start_date","day")}}/365 as admit_age
 , e.gender
 , e.race
 , hp.diagnosis_code_type as primary_diagnosis_code_type
 , hp.diagnosis_code_1 as primary_diagnosis_code
 , coalesce(icd10cm.long_description, icd9cm.long_description) as primary_diagnosis_description
-, hf.facility_id as facility_id
+, hf.facility_id as facility_ide
 , b.provider_organization_name as facility_name
 , phy.billing_id
-, {{ concat_custom(["b2.provider_first_name", "' '", "b2.provider_last_name"]) }} as provider_name
+, {{ dbt.concat(["b2.provider_first_name", "' '", "b2.provider_last_name"]) }} as provider_name
```

Comment:
```suggestion
, {{ concat_custom(["b2.provider_first_name", "' '", "b2.provider_last_name"]) }} as provider_name
```
Wondering why the change from dbt.concat from the custom macro?